.env.template

```diff
@@ -21,4 +21,6 @@ INFLUXDB_WRITE_USER_PASSWORD=write9012
 INFLUXDB_FRONTEND_ADMIN_USER=front1
 INFLUXDB_FRONTEND_ADMIN_USER_PASSWORD=front9012
 
+WORKFLOW_SETTINGS_FILE=./config/workflow_config_nwn_9sept2024.json
```

Comment:
Perhaps update the instruction in the README about this new env. workflow file path is available

Reply:
see 0f986c21b6dfe62d35b39de48d5657176acc9e19